examples/hello_world/README.md

```diff
@@ -0,0 +1,68 @@
+# Startup Script Configuration Example
+
+While in this directory, run `terraform init` then `terraform plan` and
+`terraform apply` to boot an instance which executes a custom startup script
+within the context of the standard library:
+
+# Usage
+
+```bash
+PROJECT="simple-sample-project"
+terraform apply -var project_id=$PROJECT
+```
+
+The results can be seen in the console output of the instance.
+
+```
+Jun 27 21:15:53 startup-script-stdlib-example1 startup-script: INFO startup-script: San Francisco
+Jun 27 21:15:53 startup-script-stdlib-example1 startup-script: INFO startup-script: Return code 0.
+```
+
+## Debug output
+
+```
+[jmccune@startup-script-stdlib-example1 ~]$ sudo DEBUG=1 google_metadata_script_runner --script-type startup --debug
+startup-script: INFO Starting startup scripts.
+startup-script: INFO Found startup-script in metadata.
+startup-script: INFO startup-script: 'Tue Dec 18 01:39:35 +0000 2018' Debug [5064]: init_startup_stdlib(): startup-script-stdlib.sh initialized and ready
```

Comment:
Nitpick: This nesting of `startup-script: INFO startup-script: ` is a bit funny, is that intended behavior?

Reply:
It's sort of intended.  The nesting is a result of running via the `google_metadata_script_runner` command.

Since the library will almost always run inside of the script running, I think it makes a lot of sense to cut this out of the output.

Reply:
This is a cosmetic detail, so I don't think we need to pick this over too closely - let's leave this as-is.